src/MarketingPageAcceptanceTestsSpecflow/Steps/ClientApplication/NativeMobileOrTablet/OperatingSystemsSupported.cs

```diff
@@ -0,0 +1,55 @@
+﻿using MarketingPageAcceptanceTestsSpecflow.Utils;
+using TechTalk.SpecFlow;
+
+namespace MarketingPageAcceptanceTestsSpecflow.Steps.ClientApplication.NativeMobileOrTablet
+{
+    [Binding]
+    public class OperatingSystemsSupported : TestBase
+    {
+        public OperatingSystemsSupported(UITest test, ScenarioContext context) : base(test, context)
+        {
+        }
+
+        [Given(@"that a User has provided a value for the Mandatory Information")]
+        public void GivenThatAUserHasProvidedAValueForTheMandatoryInformation()
+        {
+            _context.Pending();
```

Comment:
Why are all the steps still pending, looks like dev tasks have been completed?

Reply:
There is a bug in the front end after the rework, so waiting for that to start working again before completing the steps

Reply:
Ignore that, I forgot up update my local api...